APMrover2/commands_logic.cpp

```diff
@@ -426,6 +486,17 @@ bool Rover::verify_within_distance()
     return false;
 }
 
+// verify_yaw - return true if we have reached the desired heading
+bool Rover::verify_yaw()
+{
+    // override by do_yaw_rotation()
+    if (do_auto_rotation) {
+        return false;
```

Comment:
I'm slightly confused.  With this line won't the auto rotation go forever?  Ah you call do_yaw_rotation from  update_current_mode so that will be the continual verification that the yaw is completed.